src/css/support.js

```diff
@@ -94,9 +98,10 @@ define([
 
 		// Check for getComputedStyle so that this code is not run in IE<9.
 		if ( window.getComputedStyle ) {
-			pixelPositionVal = ( window.getComputedStyle( div, null ) || {} ).top !== "1%";
-			boxSizingReliableVal =
-				( window.getComputedStyle( div, null ) || { width: "4px" } ).width === "4px";
+			divStyle = window.getComputedStyle( div, null );
+			pixelPositionVal = ( divStyle || {} ).top !== "1%";
+			boxSizingReliableVal = ( divStyle || { height: "4px" } ).height === "4px";
```

Comment:
Hidden I frames in Firefox 4, of course: http://bugs.jquery.com/ticket/8763

I'm thinking it's safe to remove them now. :wink: